definitions/contested/json/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -10567,14 +10567,14 @@
     "CaseTypeID": "FinancialRemedyContested",
     "CaseFieldID": "interimHearings",
     "UserRole": "caseworker-divorce-financialremedy-courtadmin",
-    "CRUD": "CRUD"
+    "CRUD": "CRU"
```

Comment:
This permission I think it needs to have D - as CCD deletes and update. In past we have lot of incidents where resourcenotfound exception and it turns out Delete permission.

Reply:
do you have any reference to something similar or an explanation to this rule, because I have setup fields without D and never had issues.

Also testing this in preview worked fine.

Reply:
Yes, let me find out. @ruban] knows this